testing/libyang/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=libyang
+_srcver=0.15-r1
+pkgver=${_srcver/-r/.}
+pkgrel=0
+pkgdesc="yang parser and validator"
+url="https://github.com/CESNET/libyang"
+arch="x86_64"
+license="BSD-3-Clause"
+makedepends="cmake pcre-dev cmocka-dev bison flex doxygen"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-dbg $pkgname-utils"
+source="https://github.com/CESNET/libyang/archive/v${_srcver}.tar.gz"
+builddir="$srcdir"/$pkgname-$_srcver
+
+build() {
+	cd "$builddir"
+	mkdir build
+	cd build
+	cmake \
+		-DENABLE_BUILD_TESTS=ON \
```

Comment:
Please add the `-DENABLE_LYD_PRIV=ON` option which is not enabled by default but is required by FRR.